rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -705,18 +714,33 @@ void configure_qos_for_security(
     qos, "dds.sec.access.permissions", "permissions.p7s",
     security_options);
 
-  dds_qset_prop(qos, "dds.sec.auth.library.path", "libdds_security_auth.so");
+  dds_qset_prop(qos, "dds.sec.auth.library.path", "dds_security_auth");
   dds_qset_prop(qos, "dds.sec.auth.library.init", "init_authentication");
   dds_qset_prop(qos, "dds.sec.auth.library.finalize", "finalize_authentication");
 
-  dds_qset_prop(qos, "dds.sec.crypto.library.path", "libdds_security_crypto.so");
+  dds_qset_prop(qos, "dds.sec.crypto.library.path", "dds_security_crypto");
   dds_qset_prop(qos, "dds.sec.crypto.library.init", "init_crypto");
   dds_qset_prop(qos, "dds.sec.crypto.library.finalize", "finalize_crypto");
 
-  dds_qset_prop(qos, "dds.sec.access.library.path", "libdds_security_ac.so");
+  dds_qset_prop(qos, "dds.sec.access.library.path", "dds_security_ac");
   dds_qset_prop(qos, "dds.sec.access.library.init", "init_access_control");
   dds_qset_prop(qos, "dds.sec.access.library.finalize", "finalize_access_control");
+
+  return RMW_RET_OK;
 }
+#else
+/* Fail when security is requested but not available */
+rcutils_ret_t configure_qos_for_security(
+  dds_qos_t * qos, const rmw_node_security_options_t * security_options)
+{
+  (void) qos;
+  (void) security_options;
+  RMW_SET_ERROR_MSG(
+    "Security was requested but this Cyclone version does not have security "
+    "support enabled. Recompile with CMake option '-DDDSI_INCLUDE_SECURITY'");
```

Comment:
I think this should stay as `-DENABLE_SECURITY=ON`.

Reply:
Fixed.  Compare with [FastRTPS warning](https://github.com/ros2/rmw_fastrtps/blob/99d9178076d79e11375a4e3532db4931664b46f5/rmw_fastrtps_shared_cpp/src/rmw_node.cpp#L336)?